packages/frontend/src/components/dialogs/ViewProfile/styles.module.scss

```diff
@@ -10,10 +10,11 @@
 }
 
 .statusText {
-  padding: 15px 15px;
+  padding: 10px 15px;
   color: var(--globalText);
   white-space: pre-wrap;
   overflow-y: scroll;
+  min-height: 42px;
```

Comment:
```suggestion
  padding: 15px 15px;
  color: var(--globalText);
  white-space: pre-wrap;
```

Reply:
But that then it's always full expanded. But maybe fine here...